erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.js

```diff
@@ -15,6 +15,10 @@ frappe.ui.form.on('Plaid Settings', {
 			frm.add_custom_button('Link a new bank account', () => {
 				new erpnext.integrations.plaidLink(frm);
 			});
+
+			frm.add_custom_button(__("Sync Now"), () => {
+				frm.call("sync_now");
```

Comment:
Add a freeze

Bonus UX Task - Add a little popup (you know the one that shows on the bottom right) showing how many transactions were pulled with a button to view them.

Reply:
If it was a direct call, I could control the output, but the call is enqueued and the response isn't passed back here 😅

Reply:
Then a message saying that "It's been enqueued, please check [link to transactions list], yada yada" would work.